source/includes/administration/_quotas.md

```diff
@@ -0,0 +1,76 @@
+## Quotas
+
+Quotas are used to limit the resource usage on a service connection consumed by an organization. The primary use case for CloudMC is to enforce limited quotas on trial accounts. Quotas can target different resources. For example, number of vCPUs, allocated memory, disk size, network bandwidth, number of public IPs and more.
+
+<!------------------- LIST QUOTAS --------------------->
+
+### List quotas
+
+`GET /quotas?organization_id=:id&type=[owned|assigned]`
+
+Retrieves a list of quotas configured for a given organization. An optional `type` can be specified to retrieve either only the `assigned` or `owned` quotas.
+
+```shell
+# Retrieve product catalog list
+curl "https://cloudmc_endpoint/api/v2/quotas" \
+   -H "MC-Api-Key: your_api_key"
+```s
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "ownerOrganization": {
+        "name": "System",
+        "id": "a21debb5-b4ac-4e4b-9491-d0f4df6cd4f9"
+      },
+      "deleted": false,
+      "quotaDetails": [
+        {
+          "deleted": false,
+          "limitSize": 0,
+          "id": "1c005444-6ea2-4ebc-85f7-d75f5b1943d3",
+          "type": "INSTANCE_MEMORY",
+          "labelKey": "quotas.memory",
+          "version": 1
+        },
```

Comment:
where are the new fields, metric and ceiling?

Reply:
Good point. I checked the writer options and it's set to `*`. It's probably because I have not set any value in the DB. I will have a look.

Reply:
- [ ] Add the missing metric and ceiling fields.